community/kernel/src/main/java/org/neo4j/kernel/impl/api/StackingQueryRegistrationOperations.java

```diff
@@ -73,7 +73,7 @@ public ExecutingQuery startQueryExecution(
         String threadName = thread.getName();
         ExecutingQuery executingQuery =
                 new ExecutingQuery( queryId, clientConnection, statement.username(), queryText, queryParameters,
-                        statement.getTransaction().getMetaData(), statement.locks()::activeLockCount,
+                        statement.getTransaction().getMetaData(), () -> statement.locks().activeLockCount(),
```

Comment:
i think we have more cases where this will not gonna problems with the current statement and ExecutingQuery object, let's talk about them f2f tomorrow

Reply:
I search around a bit and couldn't find any other cases.